clients/roscpp/src/libros/spinner.cpp

```diff
@@ -130,7 +130,7 @@ void SingleThreadedSpinner::spin(CallbackQueue* queue)
 
   if (!spinner_monitor.add(queue, true))
   {
-    ROS_FATAL_STREAM("SingleThreadedSpinner: " << DEFAULT_ERROR_MESSAGE);
+    ROS_FATAL_STREAM("SingleThreadedSpinner: " << DEFAULT_ERROR_MESSAGE  << " You might want to use a MultiThreadedSpinner instead.");
```

Comment:
Looking at this critically, I think we probably want to go a bit further and update the language elsewhere.  In particular, the second sentence of `DEFAULT_ERROR_MESSAGE` doesn't seem to make much sense anymore:

```
const std::string DEFAULT_ERROR_MESSAGE =
    "Attempt to spin a callback queue from two spinners, one of them being single-threaded. "
    "This will probably result in callbacks being executed out-of-order.";
```

Since we are throwing a `std::runtime_error` directly afterwards (and thus, nothing will be happening, in-order or out-of-order).  Further, it seems like we should probably use the same string both in the `ROS_FATAL_STREAM` and as the string we use during the throw (though this string would be different depend on whether this is `SingleThreadedSpinner::spin` or `AsyncSpinnerImpl::start`.  Would you mind cleaning up those while we are doing this?  Thanks.